Makefile

```diff
@@ -143,11 +146,11 @@ releasenotes:
 
 .PHONY: verify
 verify:
-	./hack/validate-generated.sh
+	./hack/verify/validate-generated.sh
```

Comment:
wouldn't mind s/^verify-|^validate-// on all the filenames going into hack/verify